src/VisualStudio/TestUtilities2/CodeModel/Mocks/MockEditorAdaptersFactoryService.vb

```diff
@@ -0,0 +1,71 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
so, you all need this since the new type is MEF export?

Reply:
Yep, when I did the change I didn't realize I'd need to do this for the tests.  It might make more sense in this case to just do a regular instantiation.